### ISSUE TITLE
Is Brackets dead? Is it time to move on?

### ISSUE THREAD

**Issue by wolffe** [adobe/brackets#14633]
Is Brackets dead? Is it time to move on?

There hasn't been an update in a long time and the issues are piling up. Is the project dead?

**Comment by smartwebdever** [comment]
i dont think so, btw, u know that Brackets is under Adobe, and they're need time to get into all this, they have a lot of products, remember it ;) Also, you may write direct to Adobe with this question.

Have a good day, @wolffe

**Comment by wolffe** [comment]
They have a lot of products, but they also have developers. I'm looking at Atom right now which have lots of updates in a short amount of time. Brackets is not competitive. Still slow.

**Comment by ArsalanWahid** [comment]
@wolffe  its dead...  or maybe not

**Comment by Taariqq** [comment]
I would absolutely hate to see Brackets die. Visual Studio Code, or any other editor for that matter, does not have the 'live preview' capabilities that Brackets has. God no! Please, either write a plugin for VSCode that does all what Brackets does as far as live preview is concerned, or at least keep the product alive.

Question: Even if they stop updating the software we can continue to use it, correct?

Side question: I am hacking the 'keymap.json' to have 'Ctrl-L' center the page at the point of the cursor. Like it does in emacs. I have emacs keybindings installed but this key combo is not bound to center buffer.
Can anyone help?

**Comment by xypd** [comment]
Seem my worries about adobe supporting there open source product well for the long haul was unfortunately accurate. I still use it because it suits my propose for now. Not looking forward to hoping onto the next thing, which will now be electron most likely :(.

**Comment by QasimQureshi** [comment]
> Seem my worries about adobe supporting there open source product well for the long haul was unfortunately accurate. I still use it because it suits my propose for now. Not looking forward to hoping onto the next thing, which will now be electron most likely :(.

I'll have to look at Electron. Really sad to see Brackets die like this. 

Were you referring to Electronjs.org, btw?

**Comment by xypd** [comment]
I'm hoping to avoid electron if I can. So not talking about electronjs.org. No rush to switch to something else at the moment.

**Comment by shubhsnov** [comment]
Brackets is not dead. We're working on a release at the very moment which introduces the LSP framework as well as a PHPTooling extension. You can get a taste of it from the 1.14 [prerelease ](https://github.com/adobe/brackets/releases)here.

**Comment by xypd** [comment]
This is great news. Thank you @shubhsnov. It seems you are the main contributor to this project now. Appreciate your continued time you put into this. Is the beta safe to run along side the current version?

**Comment by shubhsnov** [comment]
1.14 is out now guys. And we've started work on the next release :)

**Comment by xypd** [comment]
Already downloaded and using :D. I do appreciate the follow up thanks @shubhsnov . I've also have subscribed now to brackets, so I see all the progress and open and close issues.

**Comment by MerryPanda** [comment]
it can't reload itself, lol... i'm starting to get tired of its buggery D:

**Comment by Taariqq** [comment]
> it can't reload itself, lol... i'm starting to get tired of its buggery D:
On my Linux Mint desktop, everytime I have to hit 'enter' to go to the next line, it takes a full 2-3 seconds before the cursor moves to the new line.

I 'had to' move to VSCode, but did not like it, so I settled with Apache Netbeans. The PHP workflow in Netbeans is nice!

**Comment by sylwesterdigital** [comment]
I feel like everyone else is using Visual Studio Code. The only one reason I still use brackets is that VSC is Microsoft... even worse than Adobe.

**Comment by Taariqq** [comment]
> I feel like everyone else is using Visual Studio Code. The only one reason I still use brackets is that VSC is Microsoft... even worse than Adobe.

that too, but the learning help you get when you write CSS with Brackets is unparalleled. I am learning Flexbox and love the idea of being able to see the changes being implemented instantly at each keystroke! It is nothing short of amazing.

I just hope that they keep Brackets alive for Linux. I seriously hope so.

**Comment by Taariqq** [comment]
> > it can't reload itself, lol... i'm starting to get tired of its buggery D:
> 
> On my Linux Mint desktop, everytime I have to hit 'enter' to go to the next line, it takes a full 2-3 seconds before the cursor moves to the new line.
> 
> I 'had to' move to VSCode, but did not like it, so I settled with Apache Netbeans. The PHP workflow in Netbeans is nice!

This bugginess seems to have resolved itself. Working nicely now. Thank God.
